### DISCUSSION TITLE
add reviewers field to customManagers objects

### DISCUSSION THREAD

**Discussion by ggrosseveurgg** [renovatebot/renovate#29405] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

gitlab

### Please tell us more about your question or problem

We are validating renovate changes through MRs, this works well for packageRules ... with accurate managers.

However, we need customManagers (with type regex) for some dependencies as well and the reviewers field is not supported on a customManagers block, and I cannot find any way to assign those MRs to ppl.

I tried to add a block under packageRules for same file paths but it does not seem to match with customManagers rules:
```
packageRules: [   
 {
      paths: [
        "^.platform\\/chart\\/Chart\\.yaml$"
      ],
      reviewers: ['@xxx']
    }
],
 "customManagers": [
    {
      customType: "regex",
      fileMatch: ["^.platform\\/chart\\/Chart\\.yaml$"],
      ....
    }
```

Do you have an idea how I can achieve this ?
Thanks

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by RahulGautamSingh** [comment]
You can use `matchManagers: ['custom.regex']` to match the custom managers.

**Reply by rarkins** [reply]
(in packageRules)

**Comment by ggrosseveurgg** [comment]
I've been able to try this, it doesn't seem to work as expected, the group has not been affected to the resulting MR (it works for pep621 packageRules), here's what I did:
```
packageRules: [   
 {
      paths: [
        "^.platform\\/chart\\/Chart\\.yaml$"
      ],
      matchManagers: ['custom.regex'],
      reviewers: ['@orga/core/squad-enterprise']
    }
],
"customManagers": [
    {
      customType: "regex",
      fileMatch: ["^.platform\\/chart\\/Chart\\.yaml$"],
      ....
    }
]
```
Do I miss anything ?

**Reply by rarkins** [reply]
If you have only one custom manager as in your example then you don't need any paths in your packageRules. paths is also not a valid field for packageRules either.

**Reply by ggrosseveurgg** [reply]
I have several custom managers actually, all of them should be assigned the same reviewers currently, just wanted to make it short here. 

I added each related path inside `paths` field inside my packageRules; reason I did that is in case we add more `custom.regex`, we may want to assign them to other teams using another `paths` list in another rule.

You're right, `paths` field inside my packageRules seems to be converted to `matchFileNames` according to logs, I'll update this, but this does not seem to be the issue here
